### ISSUE TITLE
Detect when drone is disconnected (powered off)

### ISSUE THREAD

**Issue by gotleg** [fetherston/npm-parrot-minidrone#4]
When the drone is turned off, we are not aware of it.
Already tried to use peripheral.on('disconnect', ...) or noble.on('disconnect', ...) with no effect.
Maybe we should try to implement some kind of ping to check that drone is alive.

**Comment by nicostuhlfauth** [comment]
I totally agree. If the drone is not powered on / available and I call new Drone() with autoconnect enabled, the program crashes.
